### ISSUE TITLE
preferred data sources to have a connection point

### ISSUE THREAD

**Issue by stijnvanhoey** [stijnvanhoey/hydropy#13]
* http://www.bafg.de/GRDC/EN/Home/homepage_node.html

**Comment by mroberge** [comment]
This is excellent!
I will try to create a system to get data from this source.

**Comment by mroberge** [comment]
Actually, it appears that the GRDC only provides data to researchers, and does not allow us to share with the general public. I am going to check to see if any of the original data sources provide their data in a way that we can use.

**Comment by mroberge** [comment]
This discussion lists several data sources by country:

https://www.researchgate.net/post/Can_anyone_tell_me_of_a_free_portal_to_download_river_discharge_data

**Comment by mroberge** [comment]
This site provides access to UK daily data as a csv file: http://nrfa.ceh.ac.uk/
Unfortunately, data sharing is not allowed.

**Comment by stijnvanhoey** [comment]
Good point of being allowed... Depends of the license information provided with the data provider. Check the description here: http://www.canadensys.net/2012/why-we-should-publish-our-data-under-cc0
